sponsors/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from users.models import CustomUser
+
+default_user = CustomUser.objects.get(email='ecell@gmail.com')
+class Sponsor(models.Model):
+	SPONS_TYPE = (
+        ('AS', 'Associate Sponsors'),
+        ('PLTS', 'Platinum Sponsors'),
+        ('GS', 'Gold Sponsors'),
+        ('TS', 'Title Sponsors'),
+        ('PRTS', 'Partner Sponsors'),
+    )
```

Comment:
Is this indentation correct, also make sure to use spaces instead of tabs and make generalized indentation settings to followed across all the app.

Reply:
Sir indentation must be correct we used auto-pep8 on this file